### PR TITLE
Add Chinese vocabularies translation

### PR DESCRIPTION
**Pull request by Bill-Haku**
translation includes:
- All of `domains.json5`
- Liyue names in `characters.json5`

### CODE REVIEW

File: dataset/dictionary/artifacts.json5 [hunk 1]
```diff
@@ -2,133 +2,133 @@
   {
     en: "artifact",
     ja: "聖遺物",
-    zh: "圣遗物",
+    zhCN: "圣遗物",
```
Comment (xicri) [inline dataset/dictionary/artifacts.json5:5]: Oh, sorry, I meant I already did this change and you didn't have to do this change.
(But you don't have to revert this change.)

File: dataset/dictionary/domains.json5 [hunk 2]
```diff
@@ -256,62 +256,74 @@
...
     tags: [ "inazuma", "domain" ],
   },
   {
     en: "Shakkei Pavilion",
     ja: "借景ノ館",
+    zhCN: "借景之馆",
     pronunciationJa: "しゃっけいノやかた",
     tags: [ "inazuma", "domain" ],
   },
   {
```
Comment (xicri) [inline dataset/dictionary/domains.json5:283]: Let's add Momiji-Dyed Court here. Can you add the Chinese translation?

```suggestion
  {
    en: "Momiji-Dyed Court",
    ja: "紅葉ノ庭",
    zhCN: "",
    pronunciationJa: "もみじノにわ",
    tags: [ "inazuma", "domain" ],
  },
  {
```

File: dataset/dictionary/domains.json5 [hunk 3]
```diff
@@ -256,62 +256,74 @@
...
     en: "End of the Oneiric Euthymia",
     ja: "夢想楽土の結末",
+    zhCN: "梦想乐土之殁",
     pronunciationJa: "むそうらくどのけつまつ",
     tags: [ "inazuma", "domain" ],
     notes: "禍津御建鳴神命戦の秘境名。",
   },
+
+  // TODO: Why there's no 椛染の庭? -- Bill Haku
```
Comment (xicri) [inline dataset/dictionary/domains.json5:328]: Oops, I didn't add Momiji-Dyed Court. I added a suggestion above.

```suggestion
```

File: dataset/dictionary/artifacts.json5 (hunk 1 above)
Comment (Bill-Haku) [reply dataset/dictionary/artifacts.json5:5]: So do I need to undo this operation? By replace all the zhCN to zh in all files.

File: dataset/dictionary/characters.json5 [hunk 4]
```diff
@@ -711,80 +732,96 @@
...
+    zhCN: "胡桃",
     pronunciationJa: "フータオ",
     tags: [ "liyue", "character-main" ],
   },
   {
     en: "Yanfei",
     ja: "煙緋",
+    zhCN: "烟绯",
     pronunciationJa: "えんひ",
+    // Pronunciataion of Chinese spelled in Japanese can be 「いぇんふぇい」, IMO. -- Bill Haku
```
Comment (xicri) [inline dataset/dictionary/characters.json5:744]: OK, let's add it as a note!

```suggestion
    notes: "英語及び中国語の読みは「イェンフェイ」",
```

File: dataset/dictionary/characters.json5 [hunk 5]
```diff
@@ -711,80 +732,96 @@
...
     pronunciationJa: "うんせんせい / うんざとう",
     tags: [ "liyue", "how-to-call" ],
     notes: "雲菫への敬称",
   },
   {
     en: "Shenhe",
     ja: "申鶴",
+    zhCN: "申鹤",
     pronunciationJa: "しんかく",
+    // Pronunciataion of Chinese spelled in Japanese can be 「しぇんへぇ」, IMO. -- Bill Haku
```
Comment (xicri) [inline dataset/dictionary/characters.json5:797]: OK, let's add it as a note!

```suggestion
    notes: "英語及び中国語の読みは「シェンヘェ」",
```

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/artifacts.json5 (hunk 1 above)
Comment (xicri) [reply dataset/dictionary/artifacts.json5:5]: No, you don't have to undo that.

File: dataset/dictionary/characters.json5 [hunk 6]
```diff
@@ -711,80 +732,96 @@
...
```
Comment (xicri) [inline dataset/dictionary/characters.json5:797]: I checked Chinese version of [Shenhe's PV](https://www.bilibili.com/video/BV1K3411e7Mk) and her name sounds シェンフゥァ to me. I checked a [Japanese website for Chinese pronunciation](https://zh.hatsuon.info/word/%E9%B9%A4) and it says 鹤 is pronounced "フゥー(ァ)" if you express it in Katakana. What do you think about it? I want your opinion.

**Comment by xicri** [comment]
@Bill-Haku
For other pronunciation comments, the Katakana expressions of Chinese pronunciation are mostly the same as English pronunciation and the users can easily read them, so I don't add them as comments this time.
However, I plan to add English (and some Chinese) pronunciation in Katakana for people who are not good at English, so I will add your Katakana expressions of Chinese pronunciation in that time.

File: dataset/dictionary/characters.json5 (hunk 6 above)
Comment (xicri) [reply dataset/dictionary/characters.json5:797]: (Sorry, I found my modification suggestion was wrong, so I rewrote this comment.)

Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:797]: the pronunciation of 鹤 in Chinese sounds like "her" in English but with less sound of "r", and there's no any sound of "f" in フ. Maybe ヘゥー(ァ) is better? What do you think about it?

**Comment by Bill-Haku** [comment]
> @Bill-Haku
> For other pronunciation comments, the Katakana expressions of Chinese pronunciation are mostly the same as English pronunciation and the users can easily read them, so I don't add them as comments this time.

You are right. That makes a sense.

> However, I plan to add English (and some Chinese) pronunciation in Katakana for people who are not good at English, so I will add your Katakana expressions of Chinese pronunciation in that time.

OK. Tell me if you need help from me.

File: dataset/dictionary/characters.json5 (hunk 6 above)
Comment (xicri) [reply dataset/dictionary/characters.json5:797]: "her" is expressed as "ハー" if you express it in Katakana. "シェンハ" might be one of the candidates? In the English version of the PV, her name sounded "シェンハ" to me.
Japanese フ is slightly different from "f" and it is also close to "h". (In Japanese we don't differentiate f and h so much) So I guess "シェンフゥァ" or  "シェンフゥー(ァ)" might be a candidate too.

"へ" is pronounced like "head"'s "he". (Not "her"'s "he") "ヘ"'s vowel is "e" but I don't think 鹤 includes the vowel of "e", so I think "シェンヘゥー(ァ)" is a bit different. (Also it is too difficult to pronounce for Japanese speakers.)

Maybe vowels are more important for Japanese speakers. We don't care about consonants so much. :laughing: (In the case of English, "s" and "th" are considered the same in Japanese. "r" and "l" are as well.)

Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:797]: I know that there are not so many phonemes in Japanese, and what I want to do is trying to express it in Japanese as close as possible to the Chinese speech, but it seems a little difficult😂. I think "シェンハ" also can be accepted.

> (Also it is too difficult to pronounce for Japanese speakers.)

That sounds true. I also think so.

Anyway, I've asked some of my friends for their opinions. I will reply to you later.

Comment (xicri) [reply dataset/dictionary/characters.json5:797]: > Anyway, I've asked some of my friends for their opinions. I will reply to you later.

OK, thanks!

Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:797]: My final recommendation is "シェンハ". 鹤 is a single-syllable word and "シェンフゥァ" or "シェンフゥー(ァ)" sounds too long and complicated.

Comment (xicri) [reply dataset/dictionary/characters.json5:797]: OK, let's use "シェンハ" here.
"シェンハ" is already written in notes, so we only have to remove the comment.

```suggestion
```

**Comment by xicri** [comment]
@Bill-Haku Thanks! I will rebase commits and merge to main branch tomorrow.

**Review by xicri — APPROVED** [review]
